app/assets/stylesheets/govuk_publishing_components/components/_layout-footer.scss

```diff
@@ -21,10 +21,43 @@
   margin-bottom: 20px;
 }
 
-// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
+  .govuk-footer {
+    background: none;
+    margin: 25mm 0 0;
+    padding: 5mm 0 0;
+    border-top: 2pt solid $govuk-print-text-colour;
+
+    .govuk-width-container {
+      margin: 0;
+    }
+  }
+
+  .govuk-footer__section-break {
+    display: none;
```

Comment:
Is there a reason that this is preferred over adding `govuk-!-display-none-print` to the HTML itself?

Reply:
Good spot thanks, I've changed it as you suggested.